rockcraft.yaml

```diff
@@ -65,3 +71,23 @@ parts:
       umount --recursive $CRAFT_OVERLAY/dev
       umount $CRAFT_OVERLAY/proc
       umount $CRAFT_OVERLAY/etc/resolv.conf
+  prometheus-exporter:
+    plugin: python
+    stage-packages:
+      - python3-venv
+    source: https://github.com/pyToshka/wazuh-prometheus-exporter.git
+    source-depth: 1
+    source-type: git
+    source-commit: 7ab31b3f73b0ce344bd7e4d0deb1d544ae1b7d1c
+    python-requirements: [requirements.txt]
+    override-build: |
+      craftctl default
+      cp -r $CRAFT_PART_SRC/* $CRAFT_PART_INSTALL/
+    organize:
+      wazuh: srv/prometheus/wazuh
+      main.py: srv/prometheus/prometheus_exporter.py
```

Comment:
we still might want to add 
```
stage:
  - srv/prometheus/wazuh
  - srv/prometheus/prometheus_exporter.py
```

Here to avoid putting unnecessary files in the rock.

Reply:
I also need the packages that are installed with the repo. Wouldn't I lose those?

Reply:
got it, if everything in `CRAFT_PART_INSTALL` is supposed to be in the rock then I think we don't need to explicitly stage.